tests/ceph_ansible/shrink_mon.py

```diff
@@ -21,6 +21,7 @@ def run(ceph_cluster, **kw):
     build = config.get("build", config.get("rhbuild"))
     mon_to_kill_list = config.get("mon-to-kill")
     mon_to_kill = None
+    cluster_name = config.get("cluster", "ceph")
```

Comment:
Now, we could use the suites to use `cluster_name` when required... if the intend is to have combination of the scenarios